src/Raven.Server/Documents/PeriodicBackup/Restore/RestoreBackupTaskBase.cs

```diff
@@ -260,6 +258,8 @@ public async Task<IOperationResult> Execute(Action<IOperationProgress> onProgres
                                 await SmugglerRestore(database, filesToRestore, context, databaseRecord, onProgress, result);
                             }
 
+                            DisableOngoingTasksIfNeeded(databaseRecord);
```

Comment:
why move it here? The tasks are enabled at this point, no?

Reply:
This is after we restored the database record but before we create the database:
https://github.com/ravendb/ravendb/blob/462b73c58d4d9d7f48081dc305e053b9cbed3efa/src/Raven.Server/Documents/PeriodicBackup/Restore/RestoreBackupTaskBase.cs#L278

Reply:
ok